app/build.gradle.kts

```diff
@@ -155,11 +158,39 @@ dependencies {
     implementation(platform(libs.firebase.bom))
 
     // Adds a remote binary dependency only for local tests.
-    testImplementation(libs.junit)
 
+    testImplementation (libs.mockito.inline)
+    testImplementation (libs.mockito.android)
+    testImplementation ("org.mockito:mockito-core:3.12.4")
+    testImplementation ("org.mockito.kotlin:mockito-kotlin:3.2.0")
+    androidTestImplementation(libs.androidx.espresso.core)
+    androidTestImplementation (libs.ui.test.junit4)
+    androidTestImplementation (libs.mockito.mockito.kotlin)
+    debugImplementation (libs.ui.test.manifest)
+    // JUnit
+    testImplementation(libs.junit)
+    androidTestImplementation(libs.junit)
 
+    // AndroidX Test
+    androidTestImplementation(libs.androidx.junit)
     androidTestImplementation(libs.androidx.espresso.core)
 
+    // Compose UI Testing
+    androidTestImplementation(libs.compose.test.junit)
+    debugImplementation(libs.compose.test.manifest)
+
+    // For both unit and instrumented tests
+    testImplementation(libs.compose.test.junit)
+    testImplementation(libs.compose.test.manifest)
+
+    // Mockito
+    testImplementation("org.mockito:mockito-core:3.12.4")
+    testImplementation("org.mockito.kotlin:mockito-kotlin:3.2.0")
+    androidTestImplementation("org.mockito:mockito-android:3.12.4")
+    androidTestImplementation("org.mockito.kotlin:mockito-kotlin:3.2.0")
```

Comment:
Warning: Having hard-coded version and libs in the build.gradle.kts is bad practice. It would be cleaner if you put them in the libs.version.toml and link them here

Reply:
Update: fixed. Changed the **JVM's** version to make it compatible